src/main/java/Lilia_Rosca/poms/LR_shopifyPage.java

```diff
@@ -23,17 +22,23 @@ public class LR_shopifyPage extends BasePage {
     public LR_PriceSection priceSection;
 
     @FindBy(css = "[data-t = color-section]")
-    public LR_PriceSection colorSection;
+    public LR_ColorSection colorSection; // modified 31.01
```

Comment:
you don't need to write comments like // modified 31.01
For this purpose we use Git